src/pybind/mgr/balancer/module.py

```diff
@@ -348,6 +351,24 @@ def show_status(self) -> Tuple[int, str, str]:
         }
         return (0, json.dumps(s, indent=4, sort_keys=True), '')
 
+    @CLIReadCommand('balancer status detail')
+    def show_status_detail(self) -> Tuple[int, str, str]:
+        """
+        Show balancer status (detailed)
+        """
+        s = {
+            'plans': list(self.plans.keys()),
+            'active': self.active,
+            'last_optimize_started': self.last_optimize_started,
+            'last_optimize_duration': self.last_optimize_duration,
+            'optimize_result': self.optimize_result,
+            'no_optimization_needed': self.no_optimization_needed,
+            'mode': self.get_module_option('mode'),
+            'added_pg_upmap_items': self.added_pg_upmap_items,
+            'removed_pg_upmap_items': self.removed_pg_upmap_items,
+            }
```

Comment:
"Make check" also failed because of this:
```
balancer/module.py:369:13: E123 closing bracket does not match indentation of opening bracket's line
```
Looks like the indentation for line 369 needs to be fixed.